gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapperTest.java

```diff
@@ -31,25 +31,55 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
+import org.slf4j.LoggerFactory;
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
 
 import java.math.BigInteger;
 import java.util.List;
 import java.util.function.Consumer;
 import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @SuppressWarnings("checkstyle:MagicNumber")
 class DuplicateObservationStatementMapperTest {
+
     private final DuplicateObservationStatementMapper mapper = new DuplicateObservationStatementMapper();
 
+    @Test
+    public void doesntMergeObservationNorDeleteItWhereObservationsAreTheSameAndHaveDifferentSameCodeableConcepts() {
```

Comment:
How is this test different to `doesntMergeObservationWhereTheCodesArentTheSame` ?

Reply:
catch catch, I will remove the duplicated test.